ARGO/include/AiStates.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
+enum AIStates { eSleeping, eWander };
+enum AITypes { eMelee, eRanged };
```

Comment:
1). What is 'e', i assume enemy but its kinda ambiguous
2). Why enum and not enum class

Reply:
1 - Coding Convection for naming enum members.